src/main/java/it/pagopa/interop/probing/eservice/operations/util/FunctionContributor.java

```diff
@@ -0,0 +1,19 @@
+package it.pagopa.interop.probing.eservice.operations.util;
+
+import org.hibernate.boot.MetadataBuilder;
+import org.hibernate.boot.spi.MetadataBuilderContributor;
+import org.hibernate.dialect.function.SQLFunctionTemplate;
+import org.hibernate.type.BooleanType;
+import org.hibernate.type.TimestampType;
+
+public class FunctionContributor implements MetadataBuilderContributor {
+
+  @Override
+  public void contribute(MetadataBuilder metadataBuilder) {
```

Comment:
Maybe it is better to use a common project for these functions, so you can share these with each project

Reply:
this is a good idea, but at the moment this is the only project that access the DB.
I would leave it here and move it to a different project when needed